ui/scripts/package.sh

```diff
@@ -3,9 +3,6 @@
 export LANG=en_US.UTF-8
 set -e
 
-npm cache clean
-bower cache clean
```

Comment:
the error described in the issue is only due to npm.

bower cache clean is still a valid command: https://bower.io/docs/api/#cache-clean

we can thus only remove the npm cache clean. just wondering, is everyone using this script likely to have npm >= 5 ?

Reply:
@teleivo . That is a decision that has been made for Bahmni Apps a while ago already so I think we can safely assume that this applies to Bahmni Connect too.
https://github.com/Bahmni/openmrs-module-bahmniapps/blob/master/ui/scripts/package.sh